integration_tests/pages/goal/ReviewUpdateGoalPage.ts

```diff
@@ -0,0 +1,34 @@
+import Page, { PageElement } from '../page'
+
+export default class UpdateGoalPage extends Page {
```

Comment:
Minor; but I think this class name should be `ReviewUpdateGoalPage` to match the filename

Reply:
Updated to `ReviewUpdateGoalPage`